src/Link.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
+
+const Link = styled.a`
+  text-decoration: none;
```

Comment:
Should this have a hover state? Maybe check with @chrisdroukas

Reply:
Update the `Link` to have default style with point cursor for the time being. Kicked off a convo with @chrisdroukas about what the style should be default hover state.

Either @chrisdroukas or I can update this default hover style, once @chrisdroukas did research to finalize his decision.

Reply:
I think an obvious one for now could be to set `text-decoration: underline` on hover. As for `cursor: pointer` that's the default for `<a>` so you don't need to set it explicitly here

Reply:
Updated per suggested. Thanks for helping to figure out what the default style of the `Link` could be for now.

Reply:
Nice! The only other thing is, should this use the default blue color from the theme?

Reply:
Theme's blue makes sense to align with the PCLN branch color. Updated the default color of `Link` component. Thanks~ ✌️